scripts/run_examples

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+
+# Run oak server
+./scripts/build_server_docker
+readonly USERNAME="$(id --user --name)"
+readonly SCRIPTS_DIR="$(dirname "$0")"
+
+docker run \
+  --detach \
+  --interactive \
+  --tty \
+  --volume="$PWD/bazel-cache:/home/$USERNAME/.cache/bazel" \
+  --volume="$PWD/cargo-cache:/home/$USERNAME/.cargo/registry" \
+  --volume="$PWD":/opt/my-project \
+  --workdir=/opt/my-project \
+  --network=host \
+  oak:latest \
+  ./bazel-bin/oak/server/oak
+SERVER_DOCKER_ID=$(docker ps -q -l)
+
+# Run oak examples
+find examples -type f -name run -exec "$SCRIPTS_DIR/docker_run" {} \;
```

Comment:
An alternative could be to have a single script that runs the server in the background and then the examples, and run the entire script in docker as a single docker command. I think this may also simplify the cleanup process. WDYT? 
Also cc @daviddrysdale